src/mapeo-project.js

```diff
@@ -492,6 +495,10 @@ export class MapeoProject extends TypedEmitter {
     return this.#dataTypes.field
   }
 
+  get translation() {
```

Comment:
I was a little surprised to see `$translation` and `translation` both be defined here. It _seems_ like we're only using it to get the type in `deleteTranslations` and in the tests. Is there a way to avoid this confusion?

Reply:
deamn. Yeah, `$translation` refers to the api, while `translation` to the `dataType`. Changing would make this the exception (which I'm not against). Some possibilities would be:
1. Make `get $translation()` into `get $translationApi()`
2. Make `get translation()` into `get translationDataType()`
3. Expose `dataType` in `translationApi` so that one can do `this.$translation.dataType`

Reply:
I think I like the last option best.